openvdb_houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -264,6 +279,38 @@ and usage examples.\n");
 }
 
 
+void
+SOP_OpenVDB_Scatter::syncNodeVersion(const char* oldVersion, const char*, bool*)
+{
+    // Since VDB 7.0.0, position compression is now set to 16-bit fixed point
+    // by default. Detect if the VDB version that this node was created with
+    // was earlier than 7.0.0 and revert back to null compression if so to
+    // prevent potentially breaking older scenes.
+
+    // VDB version string prior to 6.2.0 - "17.5.204"
+    // VDB version string since 6.2.0 - "vdb6.2.0 houdini17.5.204"
+
+    openvdb::Name oldVersionStr(oldVersion);
+
+    bool disableCompression = false;
+    size_t spacePos = oldVersionStr.find_first_of(' ');
+    if (spacePos == std::string::npos) {
+        // no space in VDB versions prior to 6.2.0
+        disableCompression = true;
+    } else if (oldVersionStr.size() > 3 && oldVersionStr.substr(0,3) == "vdb") {
+        std::string vdbVersion = oldVersionStr.substr(3,spacePos-3);
+        // disable compression in VDB version 6.2.1 or earlier
+        if (UT_String::compareVersionString(vdbVersion.c_str(), "6.2.1") < 0) {
+            disableCompression = true;
```

Comment:
I think this needs to be `<= 0` as `compareVersionString` returns 0 on version match?

Reply:
This is to handle the specific case where you've built and released an "in development" version of 6.2.1 that _doesn't_ have this change right? I think you're correct, I'll make that change.

Reply:
This is to handle all existing versions of 6.2.1 which have this node saved in it. In these cases, oldVersionStr will be 6.2.1 i.e. we don't want to change the parm interface

Reply:
Yes, that's what I understood. I've made that change now.